### ISSUE TITLE
Consider removing Jquery dependency.

### ISSUE THREAD

**Issue by tBaxter** [skratchdot/react-bootstrap-multiselect#42]
It is not needed.

**Comment by BrandonBoone** [comment]
In my bundle analysis (using this component), jQuery takes up 257.6k of my bundle size (pre minification). This is super heavy. As I am not using jQuery for anything else it's a lot of unnceessary bloat for my project. I may have some time to take a look at removing it if you'd like.

![image](https://cloud.githubusercontent.com/assets/5246918/18977248/9390802a-8686-11e6-9e91-52616a2f67d6.png)

**Comment by alexcastillo** [comment]
+1

**Comment by isaachinman** [comment]
+1

**Comment by BrandonBoone** [comment]
@isaachinman - yeah, I can take a look. I was thinking a good first pass might just be swapping out for something like [zepto.js](http://zeptojs.com/) (which is only 26kb minified). If the jQuery dependency hooks aren't super involved, then skipping to straight removal might be best. Also there's no test coverage so if someone wants to work on that it would help mitigate the risk.

**Comment by isaachinman** [comment]
I agree, a quick swap to something like zepto saves 200kb with basically zero work involved. I would assume this component doesn't do any AJAX stuff or other complex bits of jQuery.

Sadly though, I am abandoning this project nearly as quickly as I found it, after discovering the way the `onChange` function works (extremely poorly). So best of luck!

**Comment by BrandonBoone** [comment]
@isaachinman yeah I don't believe there are any AJAX dependencies so swapping out should be quick. 

So far I've avoided using `onChange` (we have an explicit "filter" button), but ping me if you find a good alternative. :-)

**Comment by isaachinman** [comment]
@BrandonBoone I ended up deciding on [react-selectize](https://www.npmjs.com/package/react-selectize) (it has a bootstrap theme). Seems to have a much nicer/in-depth API, and isn't fundamentally based on a jQuery hackjob.

I would definitely recommend it over `react-bootstrap-multiselect` to anyone reading this, past or present.
